nextcord/abc.py

```diff
@@ -1366,10 +1366,20 @@ async def send(
         if embed is not None and embeds is not None:
             raise InvalidArgument("Cannot pass both embed and embeds parameter to send()")
 
+        local_embed_files = []
         if embed is not None:
+            if embed._local_image:
```

Comment:
A lot of this logic can be simplified if we refactor the code to do:

`embeds = [embed]` and `files = [file]` early on (when `embed` or `file` are set, respectively.

The cases for single embed and single file can then be removed since the list case can cover both.

It would be similar to what is done in `interactions.py`.